src/EditorFeatures/CSharpTest/Diagnostics/GenerateMethod/GenerateMethodTests.cs

```diff
@@ -7246,7 +7246,7 @@ public void M1()
 parseOptions: TestOptions.Regular);
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateMethod)]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/15508"), Trait(Traits.Feature, Traits.Features.CodeActionsGenerateMethod)]
```

Comment:
That (not crashing) was the reason for the addition of "return false" in "CSharpTypeInferenceService.TypeInferrer.cs".

I'll adjust this test so it reflects the "broken" but non-crashing behavior.

---
In reply to: [91455957](https://github.com/dotnet/roslyn/pull/15548#discussion_r91455957) [](ancestors = 91455957)